src/js/view/controls/controlbar.js

```diff
@@ -305,7 +305,7 @@ export default class Controlbar {
 
         if (elements.cast && elements.cast.button) {
             new UI(elements.cast.element()).on('click tap enter', function(evt) {
-                if (evt.type === 'enter') {
+                if (evt.type !== 'click') {
```

Comment:
Can you leave a comment here? I understand the logic but the code reads a bit strange (if it's not a click we click?)